consensus/tests/common.rs

```diff
@@ -133,11 +133,12 @@ mod tests {
         // TODO: Validate derived state hash against ocaml derived state hash
     }
 
+    // FIXME: This test fails after state hash calculation is fixed, can we use real blocks instead?
```

Comment:
@sudo-shashank @creativcoder  Disabled this test for now, can we fix this by using real blocks instead?

Reply:
yes